gamma/utils.py

```diff
@@ -75,8 +77,6 @@ def association(picks, stations, config, event_idx0=0, method="BGMM", **kwargs):
         config["ncpu"] = min(len(unique_labels), max(1, mp.cpu_count() - 1))
 
     if len(unique_labels) == 1:
-        events = []
-        assignment = []
         event_idx = 0
         print(f"Associating {len(data)} picks with {config['ncpu']} CPUs")
         events, assignment = associate(
```

Comment:
If you pass k and labels, you would need to change this function too.

Reply:
I'll push a modification and fix the bug.